### PR TITLE
NEUBURG-59: Add version to form data

### PR DESCRIPTION
**Pull request by NKlug**

### CODE REVIEW

File: src/models/forms/neuburgForm.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
 // @flow
 
+const VERSION = 1.0
```
Comment (maxammann) [inline src/models/forms/neuburgForm.js]: Schould we make this just a number maybe? Android uses a versionCode for this which is just a incrementing counter

File: src/models/forms/neuburgForm.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+
 const MIN_TITLE_LENGTH = 10
 const MAX_TITLE_LENGTH = 50
 const MAX_LOCATION_LENGTH = 85
 
 export default {
+  // version
+  version: {
+    type: Number,
```
Comment (maxammann) [inline src/models/forms/neuburgForm.js:12]: Should ne String

**Review by maxammann — DISMISSED** [review]

File: src/models/forms/neuburgForm.js (hunk 1 above)
Comment (NKlug) [reply src/models/forms/neuburgForm.js]: Yeah we can do that. This would probably make it easier, you're right :)

Comment (NKlug) [reply src/models/forms/neuburgForm.js]: So `"1"` instead of `1.0`, right?

File: src/models/forms/neuburgForm.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // @flow
 
-const VERSION = 1.0
+const VERSION = '1'
```
Comment (maxammann) [inline src/models/forms/neuburgForm.js]: I think we need a custom form for every version so a number would be simpler. What do you think @Schedulaar ?

File: src/models/forms/neuburgForm.js (hunk 1 above)
Comment (michael-markl) [reply src/models/forms/neuburgForm.js]: Yeah.. probably a number would be easier 🤔

File: src/models/forms/neuburgForm.js (hunk 3 above)
Comment (NKlug) [reply src/models/forms/neuburgForm.js]: But then the version in the form can't be of type string:)

Comment (maxammann) [reply src/models/forms/neuburgForm.js]: yep, I just wanted to note that the type of the value and the type differ :)

**Review by michael-markl — APPROVED** [review]

**Review by steffenkleinle — APPROVED** [review]

**Pull request merged**